pallets/common/src/protocol_fee.rs

```diff
@@ -39,6 +39,7 @@ pub enum ProtocolOp {
     VotingAddBallot,
     ContractsPutCode,
     BallotAttachBallot,
+    DistributionDistribute,
```

Comment:
May be better named `CapitalDistribution`

Reply:
This was changed to `DistributionDistribute` due to https://github.com/PolymathNetwork/Polymesh/pull/716#discussion_r517797305. AIUI, the naming scheme is `ModuleExtrinsic` (just going by what @satyamakgec noted)?

Reply:
Yeah, we decided to follow the above-described naming scheme on the demand of the UI team.